feature-libs/asm/components/customer-emulation/customer-emulation.component.ts

```diff
@@ -76,20 +83,24 @@ export class CustomerEmulationComponent implements OnInit, OnDestroy {
   }
 
   openCustomer360() {
-    const data = { customer: this.customer };
-    this.launchDialogService?.openDialogAndSubscribe(
-      LAUNCH_CALLER.ASM_CUSTOMER_360,
-      this.customer360LauncherElement,
-      data
-    );
+    this.isCustomer360Loaded$
+      .pipe(filter((isReady) => Boolean(isReady)))
+      .subscribe(() => {
```

Comment:
[MEDIUM]
To avoid possible memory leaks in case when the component is very quickly destroyed, let's add this subscription to `this.subscription.add(...)` (so it can be automatically cleaned up later in `ngOnDestroy`)